### PR TITLE
notepads: Add version 1.4.2.0

### PR DESCRIPTION
**Pull request by NoraTT**

### CODE REVIEW

**Comment by NoraTT** [comment]
This is my First PR towards scoop I may of done something really stupid.

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/notepads.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    ],
+    "shortcuts": [
+        [
+            "Notepads.exe",
+            "Notepads"
+        ]
+    ],
+    "checkver": "github",
+    "autoupdate": {
+        "url": "https://github.com/JasonStein/Notepads/releases/download/v$version/Notepads_$version_x86.msixbundle#/dl.7z"
```
[the remark this answers is not in the document]
Comment (NoraTT) [reply bucket/notepads.json:34]: I copied it from a very similar install script. what do you recommend

**Comment by NoraTT** [comment]
Another thing is before we merge this this package requires a file called vcruntime140_app.dll. what vcruntime is that part of?

File: bucket/notepads.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            "pre_install": "Get-ChildItem \"$dir\" -Exclude '*x86_x64.msix' | Remove-Item -Force -Recurse"
+        },
+        "32bit": {
+            "pre_install": "Get-ChildItem \"$dir\" -Exclude '*x86.msix' | Remove-Item -Force -Recurse"
+        }
+    },
+    "installer": {
+        "script": [
+            "$winVer = [Environment]::OSVersion.Version",
+            "if (($winver.Major -lt '10') -or ($winVer.Build -lt 18362)) { throw 'At least Windows 10 18362 is required.' }",
```
[the remark this answers is not in the document]
Comment (Ash258) [reply bucket/notepads.json:19]: Next

**Pull request closed without merging**